kubernetes/structure_network_policy.go

```diff
@@ -142,23 +145,27 @@ func expandNetworkPolicySpec(in []interface{}) (*v1.NetworkPolicySpec, error) {
 func expandNetworkPolicyIngress(l []interface{}) (*[]v1.NetworkPolicyIngressRule, error) {
 	ingresses := make([]v1.NetworkPolicyIngressRule, len(l), len(l))
 	for i, ingress := range l {
-		if ingress != nil {
-			in := ingress.(map[string]interface{})
-			ingresses[i] = v1.NetworkPolicyIngressRule{}
-			if v, ok := in["ports"].([]interface{}); ok && len(v) > 0 {
-				policyPorts, err := expandNetworkPolicyPorts(v)
-				if err != nil {
-					return nil, err
-				}
-				ingresses[i].Ports = *policyPorts
+		if ingress == nil {
+			continue
+		}
+		in, ok := ingress.(map[string]interface{})
+		if !ok {
+			continue
```

Comment:
What about returning an error in that case?

Reply:
Done